index.html

```diff
@@ -153,6 +155,74 @@ <h3 class="teamName"><strong>Owca</strong></h3>
+                        <p class="additional__information--text">
+                            Na podstawie podanych informacji dobierzemy rasę kota pasującą do Ciebie.
+                        </p>
+                    </div>    
+                    <img class="additional__information--arrow" src="icons/next.svg">
+                </div>
+                <div class="additional__functionality">
+                    <div class="additional__information--desription">
+                        <img class="additional__information--icon" src="icons/search.svg">
+                        <p class="additional__information--text">
+                            Wyszukamy sprawdzonych chodowców najbliżej Ciebie.
+                        </p>
+                    </div>    
+                    <img class="additional__information--arrow" src="icons/next.svg">
+                </div>
+                <div class="additional__functionality">
+                    <div class="additional__information--desription">
+                        <img class="additional__information--icon" src="icons/food.svg">
+                        <p class="additional__information--text">
+                            Dobierzemy karmę dla wybranego przez Ciebie kota.
+                        </p>
+                    </div>    
+                    <img class="additional__information--arrow" src="icons/next.svg">
+                </div>
+                <div class="additional__functionality">
+                    <div class="additional__information--desription">
+                        <img class="additional__information--icon" src="icons/stethoscope.svg">
+                        <p class="additional__information--text">
+                            Za naszym pośrednictwem umówisz się z weterynarzem na pierwsze badanie.
+                        </p>
+                    </div>    
+                    <img class="additional__information--arrow" src="icons/next.svg">
+                </div>
+                <div class="additional__functionality">
+                    <div class="additional__information--desription">
+                        <img class="additional__information--icon" src="icons/networking.svg">
+                        <p class="additional__information--text">
+                            Uzyskasz dostęp do unikalnej bazy wiedzy dotyczącej wybranej przez Ciebie rasy.
+                        </p>
+                    </div>    
+                    <img class="additional__information--arrow" src="icons/next.svg">
+                </div>
+                <div class="additional__functionality">
+                    <div class="additional__information--desription">
+                        <img class="additional__information--icon" src="icons/volunteer_two.svg">
+                        <p class="additional__information--text">
+                            Będziesz cieszyć się wyjątkowym czasem ze swoim nowym przyjacielem.
+                        </p>
```

Comment:
dlaczego nie dodałeś tu 
    </div>    
                  <img class="additional__information--arrow" src="icons/next.svg">
   </div>

Reply:
Jak klikam na ten obrazek, to wyświetla się komunikat "There isn't anything to compare."